dojo/models.py

```diff
@@ -585,7 +585,6 @@ def findings_count(self):
             # ideally it's always prefetched and we can remove this code in the future
             self.active_finding_count = Finding.objects.filter(mitigated__isnull=True,
                                             active=True,
-                                            verified=True,
```

Comment:
is this wanted? why removing this filter?

Reply:
an active (or open) finding does not have to be verified.

Reply:
as per #1922

Reply:
ok